### DISCUSSION TITLE
Is it possible to have aDetailer run before upscaling?

### DISCUSSION THREAD

**Discussion by brknsoul** [Bing-su/adetailer#437] [Q&A]
I've enabled upscaling in "Enable additional postprocessing operations", so all my images are auto upscaled (essentially it adds the upscaling block from Process/Extras to the From Text/txt2img tab).

Is it possible to have aDetailer run _before_ the postprocessing stage?

**Comment by hikicoder** [comment]
Same need in here... Any updates?
